components/event-simulator/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/internal/CarbonEventSimulator.java

```diff
@@ -551,19 +552,26 @@ public void run() {
                 File file = new File(path);
                 FileInputStream fis = null;
                 BufferedInputStream bis = null;
-                DataInputStream dis = null;
+                BufferedReader br = null;
```

Comment:
Thanks for pointing it out. Fixed in commit - https://github.com/wso2/carbon-event-processing/pull/267/commits/d164420dea07fda6eac29b50e03ed37a3201598d